### PR TITLE
added a name

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
added a name

**Comment by sushobhana** [comment]
hi @arsh5841 ! Thanks for making this pull request. The `README.md `file is a markdown file (a particular type of format that github uses , just like html to structure your data). 
Therefore, You need to put a `-` before the name along with a whitespace to make it a bullet point. You can read more about it.
Also, you need to rebase your branch to avoid conflict. In case of any doubt, feel free to ping me!

File: README.md [hunk 1]
```diff
@@ -3,3 +3,6 @@
 ## Attendees
 
 - Sushobhana Patra (3rd year, CSE)
+Arshdeep Kumar
```
Comment (sushobhana) [inline README.md:6]: `- Arshdeep Kumar`

File: README.md [hunk 2]
```diff
@@ -3,3 +3,6 @@
...
+
+Yogesh Chaudhary
```
Comment (sushobhana) [inline README.md:8]: `- Yogesh Chaudhary`

**Review by sushobhana — CHANGES_REQUESTED** [review]

**Comment by H4wk-eye** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
NevenLeung, @john-packel, @unbalancedparentheses, @Suninus, @raineroviir, @steban83, @hapr05, @TaOliveira, @ktamas77, @mmcgraw73, @sushobhana, @yumingzhe, @pawanjay176, @moresandeep, @godwnwaswa, @nickfrost, @bensojka, @alexdwagner, @zyfrank, @erikdejonge

**Pull request closed without merging**